pkg/collector/runner/runner_test.go

```diff
@@ -33,6 +42,7 @@ func (c *TestCheck) Interval() time.Duration                            { return
 func (c *TestCheck) Run() error {
 	c.Lock()
 	defer c.Unlock()
+	defer func() { c.done <- nil }()
```

Comment:
You can just close the channel here I think

Reply:
:+1: